middleware/src/system/system.go

```diff
@@ -0,0 +1,53 @@
+//package systemenv provides functionality to get data such as open ports and services running from the system the middleware is deployed on
```

Comment:
Nit: Typo, s/`systemenv`/`system`.

Reply:
Fixed, I named the package environment, because golint complains about stuttering otherwise.

Reply:
Okay. See my other reply below for thoughts on package / struct naming.